src/components/HourGuideColumn.tsx

```diff
@@ -9,17 +9,21 @@ interface HourGuideColumnProps {
   cellHeight: number
   hour: number
   ampm: boolean
+  index: number
 }
 
-const _HourGuideColumn = ({ cellHeight, hour, ampm }: HourGuideColumnProps) => {
+const _HourGuideColumn = ({ cellHeight, hour, ampm, index }: HourGuideColumnProps) => {
   const theme = useTheme()
   const textStyle = React.useMemo(
     () => ({ color: theme.palette.gray[500], fontSize: theme.typography.xs.fontSize }),
     [theme],
   )
+  const is_pair = (i) => i % 2
```

Comment:
- is_pair should be `isPair`
- This function should be outside this component
- `(i: number)` to not be implicit `any`